### PR TITLE
Add GHI Fez Cerberus target

### PR DESCRIPTION
**Pull request by xas**
## Description

Add necessary files for creating a firmware for the Fez Cerberus (Gadgeteer board) from GHI  
Also:

* update azure pipeline
* update readme
* update PR template

## Targets affected

- [ ] BrainPad2
- [ ] GHI_FEZ_CERB40_NF
- [x] GHI_FEZ_CERBERUS_NF
- [ ] I2M_ELECTRON_NF
- [ ] I2M_OXYGEN_NF
- [ ] MBN_QUAIL
- [ ] NESHTEC_NESHNODE_V1
- [ ] NETDUINO3_WIFI
- [ ] PybStick2x
- [ ] ST_NUCLEO64_F401RE_NF
- [ ] ST_NUCLEO64_F411RE_NF
- [ ] ST_STM32F411_DISCOVERY
- [ ] ST_NUCLEO144_F412ZG_NF
- [ ] ST_NUCLEO144_F746ZG
- [ ] ST_STM32F4_DISCOVERY
- [ ] ST_NUCLEO144_F439ZI
- [ ] WEACT_F411CE
- [ ] TI_CC1352P1_LAUNCHXL_868
- [ ] TI_CC1352P1_LAUNCHXL_915
- [ ] LilygoTWatch2020
- [ ] LilygoTWatch2021
- [ ] BUILD ALL

## How Has This Been Tested?

After device flashing, tried successfully with 3 elements from gadgeteer:

* [Joystick](https://docs.ghielectronics.com/hardware/gadgeteer/modules.html#joystick)
* [Led7R](https://docs.ghielectronics.com/hardware/gadgeteer/modules.html#led-7r)
* [RFID](https://docs.ghielectronics.com/hardware/gadgeteer/modules.html#rfid-reader)

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ChibiOS/GHI_FEZ_CERBERUS_NF/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+    BOOTER_EXTRA_LINKMAP_PROPERTIES
+        ",--library-path=${CMAKE_SOURCE_DIR}/targets/ChibiOS/_common,--defsym=__main_stack_size__=0x400,--defsym=__process_stack_size__=0x400,--defsym=__crt_heap_size__=0x10000"
+
+    CLR_EXTRA_LINKMAP_PROPERTIES
+        ",--library-path=${CMAKE_SOURCE_DIR}/targets/ChibiOS/_common,--defsym=__main_stack_size__=0x400,--defsym=__process_stack_size__=0x800,--defsym=__crt_heap_size__=0x10000"
+)
+
+# if HEX2DFU tool is available pack the binaries into a DFU package
+if(HEX2DFU_TOOL_AVAILABLE)
```
Comment (josesimoes) [inline ChibiOS/GHI_FEZ_CERBERUS_NF/CMakeLists.txt:26]: You can remove this IF here.
We're not using DFU packages for some time now.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -413,6 +413,61 @@ jobs:
...
+
+  variables:
+    TargetBoard: GHI_FEZ_CERBERUS_NF
+    TargetPlatform: 'stm32'
+    TargetSeries: 'stm32f4xx'
+    BuildOptions: 
+    # creates a counter and assigns it to the revision variable
+    REVISION: $[counter('GHI_FEZ_CERBERUS_NF_versioncounter_1_8_1', 0)]
+    GccArm_Version:
+    NeedsDFU: true
```
Comment (josesimoes) [inline azure-pipelines.yml:443]: And this should be `false` (for the same reason)

File: ChibiOS/GHI_FEZ_CERBERUS_NF/resources/cerberus.pdf
Comment (Ellerbach) [inline ChibiOS/GHI_FEZ_CERBERUS_NF/resources/cerberus.pdf:1]: we're not checking in the documentations and what's under others copyright when it's not allowed.

**Review by Ellerbach — COMMENTED** [review]
All up looks really good!

**Pull request merged**